x86_energy_sync_plugin.cpp

```diff
@@ -77,7 +75,7 @@ class x86_energy_sync_metric
         // needs move and default constructor though!
         x86_energy_sync_metric(x86_energy_sync_metric&&) = default;
         x86_energy_sync_metric(const x86_energy_sync_metric&) = delete;
-        x86_energy_sync_metric& operator=(const x86_energy_sync_metric&) = delete;
+        x86_energy_sync_metric& operator=(const x86_energy_sync_metric&) = default;
```

Comment:
Still no move assignment and copy-assign suddenly default instead of delete.

`x86_energy_metric& operator=(x86_energy_metric&&) = default;`